sal_interfaces/Guider/Guider_Telemetry.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsl"?>
+<SALTelemetrySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALTelemetrySet.xsd">
+  <SALTelemetry>
+    <Subsystem>Guider</Subsystem>
+    <EFDB_Topic>Guider_offsets</EFDB_Topic>
+    <Description>Computed guider offsets. The values are computed in the instrument reference frame and need to be de-rotated accordingly by the pointing.</Description>
+    <item>
+      <EFDB_Name>x</EFDB_Name>
+      <Description>Guider offset in the (instrument) x-axis direction.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>arcsec</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>y</EFDB_Name>
+      <Description>Guider offset in the (instrument) y-axis direction.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>arcsec</Units>
+      <Count>1</Count>
```

Comment:
Please either add topic for Guider_mmOffsets specifying the offset in mm in focal plane. From mm offsets from center as measured on detector (which is independent of binning) and given guider plane rotation (and telescope F), you can convert the offset to arcsec useful for guiding. Those arcsec offsets better be specified in telescope coordinates, e.g. alt and el. You can also record mm offsets per guider (so count=8), as then you run some magic to calculate the single offset you send to the TCS.

Reply:
This is the single offset sent to the TCS... So, any magic in combining those 8 measures need to be done here. But that's a good point about using mm in focal plane.